oslc4j-client-bundle/pom.xml

```diff
@@ -0,0 +1,103 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.eclipse.lyo.clients</groupId>
+  <artifactId>oslc4j-client-bundle</artifactId>
+  <version>2.4.0-SNAPSHOT</version>
+  <packaging>bundle</packaging>
+
+
+<name>${bundle.symbolicName}</name>
+  <url>http://maven.apache.org</url>
+
+  <properties>
+    <bundle.symbolicName>oslc4j-client-bundle</bundle.symbolicName>
+    <bundle.namespace>org.oslc4j.client.bundle</bundle.namespace>
```

Comment:
I think everything we do here should begin with org.eclipse.lyo

Reply:
Agree! We need to change in oslc4j-bundle too. I was just trying to keep it in sync with that.
https://github.com/eclipse/lyo.core/blob/82ec00cf193bb2df00d45321a9d173f1126bb9ce/oslc4j-bundle/pom.xml#L19